docs/API/GraphQL-API.md

```diff
@@ -5,9 +5,11 @@ and write access to various pieces of data found in Evergreen. You can use the
 GraphQL Playground, which can be found at https://evergreen.mongodb.com/graphql,
 to experiment with and explore the available data.
 
-If you intend to use this API for production-level code, we ask that you get in
-touch with us to ensure proper support. However, if you wish to perform
-exploratory operations, there is no need to seek prior authorization.
+Please note that the Evergreen GraphQL API is currently in beta and is not
+officially supported for production-level code. If you intend to use this API
+for production, we recommend reaching out to us for proper support and
+authorization. However, for exploratory operations, you are welcome to use the
+API without prior authorization.
```

Comment:
Could be nice to highlight this a bit more
```suggestion
⚠️ Please note that the Evergreen GraphQL API is currently in beta and is not
officially supported for production-level code.

If you intend to use this API
for production, we recommend reaching out to us for proper support and
authorization. However, for exploratory operations, you are welcome to use the
API without prior authorization.
```

Reply:
Sounds good thanks!